arches_lingo/src/arches_lingo/routes.ts

```diff
@@ -35,7 +35,7 @@ export const routes = [
         component: () => import("@/arches_lingo/pages/SchemeList.vue"),
         meta: {
             shouldShowNavigation: true,
-            shouldShowHierarchy: true,
+            shouldShowHierarchy: false,
```

Comment:
I believe the mocks have this page as having the ability to open the scheme/concept hierarchy. I believe there was some discussion ( @jacobtylerwalls ? ) around having the hierarchy tree show ( or not show ) multiple schemes but I'm not sure where it landed.

<img width="387" alt="Screenshot 2024-12-03 at 2 11 51 PM" src="https://github.com/user-attachments/assets/d356446c-d584-44de-bc28-15b5d4951b80">

Reply:
Yeah I just mentioned this to @johnatawnclementawn also.

Reply:
reverted - issue should be resolved with https://github.com/archesproject/arches-lingo/issues/138